sync/class.jetpack-sync-module-posts.php

```diff
@@ -423,4 +400,68 @@ public function expand_post_ids( $args ) {
 			$this->get_term_relationships( $post_ids ),
 		);
 	}
+
+	/**
+	 * IMPORT SECTIONS .. figures out the whole importing of things.
+	 */
+	public function sync_import_done( $importer ) {
```

Comment:
It doesn't have to be in this PR, but lets see if we can get this import business into a file/module of its own in the near future.